meetings/http/2016-11-21/minutes.md

```diff
@@ -1,19 +1,63 @@
 # HTTP sub-team kick-off
 
-### Time: Monday Nov 21st, 2016
-* 7pm CET 
-* 6pm UK 
-* 1pm EST 
-* 10am PST
-
-### WebEx and Agenda/Minutes doc:
-* WebEx:     https://meetings.webex.com/collabs/#/meetings/detail?uuid=M8C65H48UFLXGRIS8N2GVWH0DV-1T5J&rnd=78810.922722
-* Agenda:    https://docs.google.com/document/d/1SWK0qBDi-9DeLJwHlcXcPU7h22JU-DWW8HTVuHbTQiw/edit?usp=sharing
-
-If you'd like to add anything to the agenda ahead of time, please update the agenda document in Google Docs.
-
 ### Initial Agenda:
 * Introductions
 * Goals
 * Development process overview
 * Next Steps
+
+### Attendees:
+* Chris Bailey
+* Alex Blewitt
+* Ryan Collins
+* Ben Cohen
+* Sam Liu
+* Shmuel Kallner
+* Tyler Stromberg
+* Robert F. Dickerson
+* Logan Wright
+* Tanner Nelson
+* David Sperling
+* Kuan Huang
+* Johannes Weiss
+
+### Minutes:
+
+_How are we going to interface these things together, ie, how HTTP sits on networking?_
+* Concurrency between now and at least Swift 5 is going to be Dispatch based. From Swift 5 and onwards there may be/will be a new concurrency model. We’ll need to transition at that time.
```

Comment:
the will be is unnecessary here, since it is covered by the may be. In any case the high level goals for Swift 5 have not yet been debated, let alone agreed, so it would be incorrect to conclusively state that they are part of it.